hvm/cmd/install.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```

Comment:
```suggestion
```

Reply:
Ugh. Yeah. Those keep getting added by the Cobra cli tool. I'll remove them. I got a bunch, but guess I missed a few. Thanks.